demisto_sdk/commands/content_graph/content_graph_commands.py

```diff
@@ -55,7 +55,10 @@ def update_content_graph(
         output_path (Path): The path to export the graph zip to.
     """
     if use_git and imported_path:
-        raise ValueError("Cannot use both git and imported path")
+        logger.debug(
+            "The imported path will be ignored since we are using git to get the packs to update."
```

Comment:
is it ignored? I thought it's used
If I'm not mistaken, it should be a WARNING log, something like "Overriding existing graph exported files"